### ISSUE TITLE
It's been a long time.

### ISSUE THREAD

**Issue by developer0hye** [yjh0410/YOLOX-Backbone#1]
Hi @yjh0410 

How are you?

I made a similar project like this project. https://github.com/developer0hye/YOLOX-Backbone

I just came here to say hello.

**Comment by yjh0410** [comment]
Hi ！Long time no see @developer0hye 

Everything is fine.

You really did an excellent job, I should have found out sooner.

**Comment by developer0hye** [comment]
@yjh0410 
Thanks! It seems that we want to develop similar things.
Are you still working on your phd?

**Comment by yjh0410** [comment]
@developer0hye 
Yeah, I am now in the third year of my Ph.D., and it will take about 2 years for me to graduate. A PhD in China usually takes 4 to 5 years to graduate. I was in my first year of Ph.D. when we first met.

**Comment by developer0hye** [comment]
@yjh0410 
Great! I have just opened my work on object detection!

https://github.com/MarkAny-Vision-AI/CenterNeXt

You seem to be interested in this project.

**Comment by yjh0410** [comment]
@developer0hye Excellent! I am really interested in your CenterNeXt. This is a awesome name!

Looking forward to releasing the weight of COCO in the future.

**Comment by developer0hye** [comment]
@yjh0410 
Okay! Thanks!

**Comment by yjh0410** [comment]
@developer0hye Hi!I have a question about the training configuration files. I noticed that the epoch is set to 70，which is less than official centernet's, so, only 70 epoch is enough?

**Comment by developer0hye** [comment]
@yjh0410 
70epoch is enough to train a model on VOC dataset.
Howerver, it is not enough to train a model on COCO dataset. I got 24 AP on COCO dataset with this setting.

Do you have any advice for me?

 I will train a model for 140 epochs on COCO dataset.

**Comment by developer0hye** [comment]
https://github.com/MarkAny-Vision-AI/CenterNeXt/tree/coco-dataloader

I am working on the above branch.
